README.md

```diff
@@ -79,8 +79,11 @@ job2 = {"runs": 50, "variables": {"omega_max": 10.5} }
 # Send the batch of jobs to the QPU and wait for the results
 batch = sdk.create_batch(serialized_sequence, [job1,job2], wait=True)
 
-# You can also choose to run your batch on an emulator using the optional argument 'emulator'
-# batch = sdk.create_batch(serialized_sequence, [job1,job2], emulator=True)
+# You can also choose to run your batch on an emulator using the optional argument 'device_type'
+# For using a basic single-threaded QPU emulator that can go up to 10 qubits, you can the "EMU_FREE" device type.
+# "EMU_SV", which is a more performant version, is not available on this platform yet.
```

Comment:
Didn't we decide in the end to just keep EMU_FREE for now?

Reply:
I think SV is here also for purposes of the end to end tests

but the performant version should go live soon enough